docs/content/dagster-cloud/developing-testing/environment-variables-and-secrets.mdx

```diff
@@ -44,29 +29,53 @@ In the following table, we compare the supported methods for managing and passin
 >
   <thead>
     <tr>
+      <th></th>
       <th
         style={{
-          width: "15%",
+          width: "40%",
         }}
       >
-        Setup method
+        Dagster Cloud UI
       </th>
       <th
         style={{
           width: "40%",
         }}
       >
-        How it works
+        Agent configuration
       </th>
-      <th>May be a good fit if...</th>
     </tr>
   </thead>
   <tbody>
     <tr>
-      <td>GitHub secrets</td>
       <td>
-        Environment variables are defined in a GitHub repository using Action
-        Secrets. <strong>Supported for Serverless deployments only</strong>.
+        <strong>Deployment type support</strong>
+      </td>
+      <td>
+        <a href="/dagster-cloud/deployment/serverless">Serverless</a>,{" "}
+        <a href="/dagster-cloud/deployment/hybrid">Hybrid</a>
+      </td>
+      <td>
+        <a href="/dagster-cloud/deployment/hybrid">Hybrid</a>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>How it works</strong>
+      </td>
+      <td>
+        Environment variables are managed in the Dagster Cloud UI. Values are
+        pulled from storage and decrypted at run time.
+      </td>
+      <td>
+        Environment variables are defined in the agent's configuration in your
+        project's <code>dagster_cloud.yaml</code> file. Values are pulled from
```

Comment:
I think the question here is do we still want to make this option an obvious choice in these docs. It feels much like the GH secrets method, in that it's more steps and more complicated and more likely to trip users up.

We **could** plunk the Hybrid env vars stuff in its own guide to de-emphasize it and then link to it from here. Thoughts? I feel like having it in here muddies the UI story and adds unnecessary complexity. Additionally, if we're saying the UI is the recommended way of doing things, I feel comfortable doing this.

Reply:
I think there are two different things here:
- Platform-specific secrets like kubernetes secrets or aws secretsmanager - it seems important that we have clear instructions for people who don't want to send us their secret values, so I think we need to keep that part
- Hybrid env vars where you give us the actual key-value pairs in container_context (i'm fine leaving this out since it's not any better than the new UI)

Reply:
As per slack convo i think we can link to the hybrid agent docs here for now